core/baseline-core-v1.0-psd01.md

```diff
@@ -819,65 +825,100 @@ The identifiers and identity utilized in a BPI MUST be controlled by its Princip
 
+[[CR2]>[D11]](#cr2>d11) Testability: Verification to determine that a set of keys associated with a DID in a DPKI is the valid and latest set of keys can be achieved by:
+1. Using a universal resolver to get the latest set of keys associated with the DID used in a BPI
+2. Extracting the block and transaction hash of the latest update on the CCSM used by the DID method from the metadata of the universal resolver.
+3. Verifying if the latest set of keys are in the longest chain using a block explorer such as [Etherscan](https://etherscan.io/).
+
 *Requirements of Identifier registration in DPKI are handled differently from DNS. Although registrars may exist in DPKI, these registrars must adhere to several requirements that ensure that identities belong to the entities they represent. This is achieved the following way:*
 
 #### **[CR3]>[D11]**	
 Private keys utilized in a BPI MUST be generated in a manner that ensures they remain under the Principal Owner’s control. 
 
+[[CR3]>[D11]](#cr3>d11) Testability: To ensure the Principal Owner is solely reponsible for the creation and use the set of public/private keys associated with their DID, a non-custodial wallet must be used to initiate all transactions on behalf of that principal owner within a BPI. An example of implementing this would be to integrate this non-custodial wallet with the BPI APIs that ensure all API payloads/messages are signed, and therefore authorized, solely by the Principal Owner's private key before executing BPI transactions.
+
 #### **[CR4]>[D11]** 
 Generating key pairs in a BPI on behalf of the Principal Owner MUST NOT be allowed.
 
+[[CR4]>[D11]](#cr4>d11) Testability: An example of cutting off the ability to generate public/private key pairs for a subject is by ensuring there are no custodial wallets that exist within a BPI.
+
 #### **[CR5]>[D11]**	
 Principal Owners in a BPI MUST always be in control of their identifiers and the corresponding public keys. 
 
+[[CR5]>[D11]](#cr5>d11) Testability: To guarantee Principal Owners are the sole controller over their identifiers and related public keys, validation can be conducted in the same manner as [[CR2]>[D11]](#cr2>d11). 
+
 #### **[O1]**	
 Principal Owners MAY extend control of their identifier to third parties.
 
 *For example for recovery purposes.*
 
+[[O1]](#o1) Testability: An implementer can verify whether a principal owner has delegated authority of their identifier to third parties by checking in the DID document if there are other DIDs in the [controller](https://www.w3.org/TR/did-core/#did-controller) section of the did document and what type of control these identifiers have by checking the [verification relationships](https://www.w3.org/TR/did-core/#verification-relationships) such as authentication or signing on behalf of [assertion](https://www.w3.org/TR/did-core/#assertion) in the DID document.
```

Comment:
I don't think you can test for this, except as a process. I can do as outlined, but I can also just write the info in a christmas card for my Mum and tell her to use it wisely.